react/LocationSearch.tsx

```diff
@@ -100,7 +101,11 @@ const LocationSearch: React.FC<LocationSearchProps> = ({
             <>
               <ComboboxList>
                 {addresses.map((address, index) => (
-                  <ComboboxOption value={address} key={index} />
+                  <ComboboxOption
+                    renderIcon={() => <PlaceIcon />}
```

Comment:
idk, I have mixed feelings about this. @lucasecdb when we discussed, I imagined this implementation as `icon={PlaceIcon}`, but:
- somehow it wasn't working;
- as I don't really see the benefit by picking one or the other (in that case), I decided to render it like that.

Reply:
I was thinking more on a `icon={<PlaceIcon />}` to be honest..